build/Maestro/Maestro.csproj

```diff
@@ -0,0 +1,49 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- TFM doesn't matter. These settings are required to make NuGet happy so we can restore required MSBuild packages. -->
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <ManifestsPath>$(ArtifactsDir)manifests\</ManifestsPath>
+    <DisablePackageReferenceRestrictions>true</DisablePackageReferenceRestrictions>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.Maestro.Tasks" Version="1.1.0-beta.19065.2" />
+  </ItemGroup>
+
+  <Target Name="GetFilesToPublish">
+    <ItemGroup>
+      <PackageToPublish Include="$(ArtifactsShippingPackagesDir)*.symbols.nupkg" IsSymbolsPackage="true" />
+      <PackageToPublish Include="$(ArtifactsShippingPackagesDir)*.nupkg" Exclude="@(PackageToPublish)" />
```

Comment:
Excluding `$(ArtifactsShippingPackagesDir)*.symbols.nupkg` would be more obvious.

Reply:
Oops, should have said "and switching the two lines".

Reply:
:up: :date:

This file was the result of deleting obsolete code from publish.csproj and renaming the file. Now that I'm looking at this, I see I can delete the `IsSymbolsPackage` attribute too. There should be no reason anymore to single out the `.symbols.nupkg` files.